cmd/conduit/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/algorand/indexer/conduit"
+	"github.com/algorand/indexer/config"
+)
+
+import (
+	_ "github.com/algorand/indexer/exporters/all"
+	_ "github.com/algorand/indexer/importers/all"
+	_ "github.com/algorand/indexer/processors/all"
+)
+
+var (
+	logger          *log.Logger
+	defaultLogLevel string
+)
+
+// init() function for main package
+func init() {
+
+	// Setup logger
+	logger = log.New()
+
+	formatter := conduit.PluginLogFormatter{
+		Formatter: &log.JSONFormatter{
+			DisableHTMLEscape: true,
+		},
+		Type: "Conduit",
+		Name: "main",
+	}
+
+	logger.SetFormatter(&formatter)
+	logger.SetOutput(os.Stdout)
+	defaultLogLevel = "info"
+}
+
+// conduitCmd creates the main cobra command, initializes flags, and viper aliases
+func conduitCmd() *cobra.Command {
+	cfg := &conduit.Config{}
+	conduitCmd := &cobra.Command{
+		Use:   "conduit",
+		Short: "run the conduit framework",
+		Long:  "run the conduit framework",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Could you have a `Run` function which accepts the config object? We'll hopefully get to use it later in `daemon.go`.